### PR TITLE
Edited

### PR DESCRIPTION
**Pull request by Paryax**
- [DEMO LINK](https://Paryax.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Paryax.github.io/layout_moyo-header/report/html_report/)

Fixed links

### CODE REVIEW

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
Let's add link to description

File: src/style.css [hunk 1]
```diff
@@ -1 +1,81 @@
...
+body {
+  margin: 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 0;
```
Comment (Ihor-Karpyn) [inline src/style.css:18]: redundant remove it

File: src/style.css [hunk 2]
```diff
@@ -1 +1,81 @@
...
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  margin: 0;
+  display: flex;
+  padding: 0;
```
Comment (Ihor-Karpyn) [inline src/style.css:33]: Reset all margin and padding which u need like we did it on the lesson

```
ul,
p,
... {
  mar: 0
  pad: 0
}
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,81 @@
...
+  margin: 0;
+}
+
+.nav_link {
+  padding-top: 23px;
+  padding-bottom: 23px;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  height: 15px;
```
Comment (Ihor-Karpyn) [inline src/style.css:55]:
```suggestion
  height: 60px;
```

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
Let's add data-qa="hover", check readme, try to fix the active line
image

File: src/style.css (hunk 2 above)
Comment (ivan-kadykalo) [reply src/style.css:35]: Fix Ihor's comment

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
1. Check readme and add the missing [data-qa="hover"] attribute
image

2. Fix the space between the link and the bottom line

**Review by ivan-kadykalo — APPROVED** [review]
GJ🚀

**Pull request closed without merging**